views/default/input/date.php

```diff
@@ -48,8 +52,20 @@
 	$vars['value'] = gmdate('Y-m-d', $vars['value']);
 }
 
+$datepicker_options = elgg_extract('datepicker_options', $vars);
```

Comment:
Only other thing we probably want is to move logic to elgg.ui_initDatePicker (`@private` in jsdoc), call that from core. elgg.ui.initDatePicker would just show deprecation notice and call the private one.